src/probnum/linalg/solvers/policies/_conjugate_gradient.py

```diff
@@ -0,0 +1,31 @@
+"""Policy returning :math:`A`-conjugate actions."""
+
+import numpy as np
+
+import probnum  # pylint: disable="unused-import"
```

Comment:
Why do we need this here?

Reply:
We need `probnum.linalg.solvers.LinearSolverState` as a type hint for the policies. Due to circular dependencies we need to use a forward references. However, Sphinx can only resolve this if the corresponding module is actually imported.